dali/sasha/sacoalescer.cpp

```diff
@@ -177,7 +177,7 @@ class CSashaSDSCoalescingServer: public ISashaServer, public Thread
     CSashaSDSCoalescingServer(IPropertyTree *_config)
         : Thread("CSashaSDSCoalescingServer"), coalesceProps(_config)
     {
-        stopped = false;
+        stopped.store(false);
```

Comment:
comment only: not strictly necessary to change this/other lines. stopped = false is equivalent to stopped.store(false)